### PR TITLE
www-client/chromium: add missing licenses, drop unrar dependency

### PR DESCRIPTION
**Pull request by scardracs**
For tracking. If any dev wants to overtake it feel free to do so.
Bug https://bugs.gentoo.org/492424

Signed-off-by: Marco Scardovi <mscardovi@icloud.com>

### CODE REVIEW

File: www-client/chromium/chromium-108.0.5343.2-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,1228 @@
...
+		third_party/swiftshader/third_party/SPIRV-Headers/include/spirv
+		third_party/swiftshader/third_party/SPIRV-Tools
+		third_party/tensorflow-text
+		third_party/tflite
+		third_party/tflite/src/third_party/eigen3
+		third_party/tflite/src/third_party/fft2d
+		third_party/ruy
+		third_party/six
+		third_party/ukey2
+		third_party/unrar
```
Comment (stikonas) [inline www-client/chromium/chromium-108.0.5343.2-r1.ebuild:560]: Is this line still needed?

File: www-client/chromium/chromium-107.0.5304.121-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,1230 @@
...
+PATCHSET_NAME="chromium-$(ver_cut 1)-patchset-${PATCHSET}"
+PATCHSET_NAME_PPC64="chromium_107.0.5304.68-1raptor1~deb11u1.debian"
+SRC_URI="
+	https://commondatastorage.googleapis.com/chromium-browser-official/${P}.tar.xz
+	https://github.com/stha09/chromium-patches/releases/download/${PATCHSET_NAME}/${PATCHSET_NAME}.tar.xz
+	ppc64? ( https://quickbuild.io/~raptor-engineering-public/+archive/ubuntu/chromium/+files/${PATCHSET_NAME_PPC64}.tar.xz )
+	pgo? ( https://github.com/elkablo/chromium-profiler/releases/download/v0.2/chromium-profiler-0.2.tar )
+"
+
+LICENSE="BSD Apache-2.0 APSL-2 BSD-2 GPL-2 GPL-2+ GPL-3+ icu ISC LGPL-2 LGPL-2+ LGPL-2.1 LGPL-2.1+ MIT MPL-1.1 MPL-2.0 Ms-PL public-domain"
```
Comment (stha09) [inline www-client/chromium/chromium-107.0.5304.121-r1.ebuild:29]: ICU is conditional with USE=-system-icu

**Comment by stha09** [comment]
Honestly I don't merge this with removing unrar code, because you open a security risk for users without notice.

**Comment by stikonas** [comment]
> Honestly I don't merge this with removing unrar code, because you open a security risk for users without notice.

Then the other option is to make package masked by default and add LICENSE="unrar". This would also upset people.

Perhaps patch should be conditional on USE flag?

File: www-client/chromium/chromium-108.0.5343.2-r1.ebuild (hunk 1 above)
Comment (scardracs) [reply www-client/chromium/chromium-108.0.5343.2-r1.ebuild:560]: We are looking if maintains it and "enforce" people to accept unRAR license or to make it conditional

**Comment by scardracs** [comment]
Closing as floppym is overtaking it.

**Pull request closed without merging**